### DISCUSSION TITLE
cannot get pull request

### DISCUSSION THREAD

**Discussion by rsarosh** [community/community#24452] [Apps, API and Webhooks]
<p>I am admin on the repository xxx, but i cannot list all the pull request for the repository.</p>
<p><a href="https://api.github.com/repos/rsarosh/xxx/pulls?state=all" rel="nofollow noopener">https://api.github.com/repos/rsarosh/xxx/pulls?state=all</a></p>
<p>Reponse gives me 404. What am i missing?</p>
<p>However, this API works : <a href="https://api.github.com/user/repos?type=all&amp;per_page=1000" rel="nofollow noopener">https://api.github.com/user/repos?type=all&amp;per_page=1000</a></p>

**Comment by den3tsou** [comment, marked as answer]
<p>Do you create the personal toke to have a link like this <a href="https://api.github.com/repos/rsarosh/xxx/pulls?state=all&amp;access_token=xxxxx" rel="nofollow noopener">https://api.github.com/repos/rsarosh/xxx/pulls?state=all&amp;access_token=xxxxx</a>. You should be able to get it in settings -&gt; developer settings -&gt; personal token.</p>
